go/runtime/bundle/manifest.go

```diff
@@ -41,3 +94,41 @@ type SGXMetadata struct {
 	// Signature is the name of the SGX enclave signature file.
 	Signature string `json:"signature"`
 }
+
+// ComponentKind is the kind of a component.
+type ComponentKind string
```

Comment:
I used a string since this is most commonly serialized into JSON. We could also use an `uint8` with custom serialization/deserialization but it just adds some more code.